modules/scf/gen_config.sh

```diff
@@ -18,6 +18,8 @@ if [ "$SCF_OPERATOR" != true ]; then
     fi
 fi
 
+kubectl patch -n kube-system configmap cap-values -p $'data:\n services: "'$SCF_SERVICES'"'
+services="$SCF_SERVICES"
 domain=$(kubectl get configmap -n kube-system cap-values -o json | jq -r '.data["domain"]')
 public_ip=$(kubectl get configmap -n kube-system cap-values -o json | jq -r '.data["public-ip"]')
```

Comment:
again, Public ip should not be required for public clouds, AFAIK

Reply:
(see other comment)